src/app/public/modules/date-range-picker/types/date-range-format-type.ts

```diff
@@ -0,0 +1,24 @@
+export enum SkyDateRangeFormatType {
+  SpecificRange = 0,
```

Comment:
Enums can be auto-incremented with this format:
```
export enum SkyDateRangeFormatType {
  SpecificRange = 0,
  Before,
  After,
  // ...
}
```

Reply:
done